src/winetricks

```diff
@@ -718,6 +718,15 @@ w_expand_env()
     winetricks_early_wine cmd.exe /c echo "%$1%"
 }
 
+# Check if $1 is executable (output custom message using w_debug if $2 is specified)
```

Comment:
$1/$2 should be assigned to variables.

Reply:
resolved in https://github.com/Winetricks/winetricks/commit/2084d42a5179006805c1790722073ff43ceabf91